specification/monitor/resource-manager/Microsoft.Insights/preview/2023-06-01-preview/privateLinkScopes_API.json

```diff
@@ -0,0 +1,1110 @@
+              "$ref": "#/definitions/AzureMonitorPrivateLinkScope"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateLinkScopeCreate": {
+            "$ref": "./examples/PrivateLinkScopesCreate.json"
+          },
+          "PrivateLinkScopeUpdate": {
+            "$ref": "./examples/PrivateLinkScopesUpdate.json"
+          }
+        }
+      },
+      "patch": {
+        "description": "Updates an existing PrivateLinkScope's tags. To update other fields use the CreateOrUpdate method.",
+        "operationId": "PrivateLinkScopes_UpdateTags",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateLinkScopeName"
+          },
+          {
+            "name": "PrivateLinkScopeTags",
+            "description": "Updated tag information to set into the PrivateLinkScope instance.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TagsResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updating the Azure Monitor PrivateLinkScope's tags was successful. PrivateLinkScope tags are updated and returned with the rest of the PrivateLinkScope's object properties.",
+            "schema": {
+              "$ref": "#/definitions/AzureMonitorPrivateLinkScope"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateLinkScopeUpdateTagsOnly": {
+            "$ref": "./examples/PrivateLinkScopesUpdateTagsOnly.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Insights/privateLinkScopeOperationStatuses/{asyncOperationId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get specific operation status": {
+            "$ref": "./examples/privateLinkScopeOperationStatuses.json"
+          }
+        },
+        "tags": [
+          "asyncOperations"
+        ],
+        "operationId": "PrivateLinkScopeOperationStatus_Get",
```

Comment:
>PrivateLinkScopeOperationStatus_Get [](http://example.com/codeflow?start=24&length=35)

Is this intended as an api consistency fix? 

I'm not sure whether we consider fixing this best practice or not.

I think you might get breaking change errors from having *reordered* the parameters here.

Reply:
I'm not entirely sure what the operation_Id is intended for, but I will mention that this was part of the existing base commit - seems to be a common pattern across the file.

Reply:
Please temporarily ignore this feedback from me as it can be covered by SDK breaking changes review process, calling this resolved for now. @qinjoshua